src/web/modules/transactions/update/update.http.ts

```diff
@@ -49,18 +51,24 @@ const validateAndAddDefaults = async function validateAndAddDefaults(csv: string
         if (!row.event_date) {
           row.event_date = moment().utc().format()
         }
+        if (!row.transaction_type) {
+          row.transaction_type = 'PAYMENT'
```

Comment:
Looking at https://github.com/alphagov/pay-ledger/blob/2a4a0ed3edab80d4282449dade32d417b60d887f/src/test/java/uk/gov/pay/ledger/queue/EventQueueTest.java I think Ledger might actually expect this lower case (even if the DB enums are then upper case) -- so the default and README should use `payment` and `refund` unless we want to transform those too

Reply:
yeh makes sense to make lower case then